cg/models/validation_cases/validation_case_data.py

```diff
@@ -0,0 +1,118 @@
+"""Model for down sampling meta data."""
+
+import logging
+from pathlib import Path
+
+from cg.apps.housekeeper.hk import HousekeeperAPI
+from cg.constants import DataDelivery, Priority
+from cg.store.models import ApplicationVersion, Case, Sample
+from cg.store.store import Store
+from cg.utils.calculations import multiply_by_million
+
+LOG = logging.getLogger(__name__)
+
+
+class ValidationCaseData:
+    def __init__(
+        self,
+        status_db: Store,
+        hk_api: HousekeeperAPI,
```

Comment:
I don't think we should have the DB APIs here. Models are a way to encapsulate data, and provide methods to work with that data, but we should extract it in the CreateValidationCase API. Then you could also make use of a pydantic models here to perform all the validations


```suggestion
```

Reply:
I moved it out of the models as the class does not represent a typical model class indeed.